src/internal/components/chart-popover/index.tsx

```diff
@@ -31,6 +33,7 @@ interface ChartPopoverProps extends PopoverProps {
     </>)
   */
   trackKey?: string | number;
+  minHeight?: number;
```

Comment:
Do we want to change this name here accordingly as well now that we have the chance? Doing so later once chart-components is already using this might be harder